Deferred/DispatchBlockFlags.swift

```diff
@@ -10,24 +10,29 @@ import Dispatch
 
 extension dispatch_block_flags_t: RawOptionSetType {
 
+    /// The empty options set.
     public static var allZeros: dispatch_block_flags_t {
         return dispatch_block_flags_t(0)
     }
 
+    /// Convert from a bitmask of `UInt`.
     public init(rawValue: UInt) {
         self.init(rawValue)
     }
 
+    /// Create an instance initialized with `nil`.
```

Comment:
I don't think it's necessary for all documentation to be in line with the Swift docs, we should just be consistent on our own and ideally fit some shade of BNR voice.